test/e2e/scenarios/Cluster/cluster_scale_in_scale_up_increase_replica.yaml

```diff
@@ -28,7 +28,7 @@ steps:
         conditions:
           ACK.ResourceSynced:
             status: "True"
-            timeout: 7200
+            timeout: 5400
```

Comment:
Could you add to the `description` above why these timeouts are so long? Explain about the snapshot window problem and why we can't just change the window to get around it

Reply:
Because we never know when we run e2e tests ...

Reply:
Yes we can change snapshot window to any time, but if we run e2e tests in that specific time range, then it is going to automatically snapshot.

Reply:
It is for all clusters, so I guess we don't need to add description for each one?

Reply:
Why can't we make the snapshot window a dynamic variable? That way we can set it to be 12 hours apart from the time we start the tests?

Reply:
Do we have a way to set a dynamic variable in yaml file? Actually I don't know how to do it. Also, by default, If we do not specify this parameter, MemoryDB automatically chooses an appropriate time range.

Reply:
Will create a new PR to fix Snapshot Window issue.

Reply:
Update: `SnapshotWindow` issue is fixed in PR #66. Ideally cluster should finish creation within 30 minutes. However, I find that k8s needs to spend some time to finish `synced` after operations of a resource is done in aws service. In this case, it is rarely that k8s couldn't finish `synced` within 30 minutes after a cluster starts creating. Hence, I set all wait time for cluster and snapshot to 3600s to make sure they have enough time.

Reply:
This will be because we do an exponential backoff while waiting for updates. Our backoff window might grow past 30 minutes. 3600s is fine for that, then